tools/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ target_link_libraries(rpmlua PRIVATE LUA::LUA)
 target_link_libraries(rpmbuild PRIVATE librpmbuild)
 target_link_libraries(rpmspec PRIVATE librpmbuild)
 target_link_libraries(rpmdeps PRIVATE librpmbuild)
+target_link_libraries(rpmuncompress PRIVATE archive)
```

Comment:
Is this the name of the dependency? It looks like it's `LIBARCHIVE` instead of `archive`.

Reply:
With `LIBARCHIVE` I get:
```
[ 77%] Linking C executable rpmuncompress
/usr/bin/ld: cannot find -lLIBARCHIVE: No such file or directory
collect2: error: ld returned 1 exit status
```

Reply:
Follow the rpm2archive.c example:

`target_link_libraries(rpm2archive PRIVATE PkgConfig::LIBARCHIVE)`

That covers other aspects besides the library linkage name.